cpp/mrc/src/internal/data_plane/data_plane_resources.cpp

```diff
@@ -160,6 +161,20 @@ DataPlaneResources2::DataPlaneResources2()
     DVLOG(10) << "initialize the registration cache for this context";
     m_registration_cache = std::make_shared<ucx::RegistrationCache2>(m_context);
 
+    auto decrement_callback = ucxx::AmReceiverCallbackType([this](std::shared_ptr<ucxx::Request> req) {
+        if (req->getStatus() != UCS_OK)
+            // TODO(Peter): Ensure the error gets raised somehow
+            DVLOG(10) << "Error calling decrement_callback";
```

Comment:
Our linter prefers brackets `{}` around all statements. Even one line for/if statements